data/data/openstack/variables-openstack.tf

```diff
@@ -341,16 +353,52 @@ variable "openstack_master_server_group_policy" {
   description = "Policy of the server group for the master nodes."
 }
 
-variable "openstack_machines_subnet_id" {
-  type        = string
-  default     = ""
-  description = "ID of the subnet to use for cluster machines. If empty, the installer will create a subnet to use as machinesSubnet."
-}
-
-variable "openstack_machines_network_id" {
-  type        = string
-  default     = ""
-  description = "ID of the network the machines subnet is on. If empty, the installer will create a network to use as machinesNetwork."
+/* variable "openstack_default_machines_subnet_id" { */
+/*   type        = string */
+/*   default     = "" */
+/*   description = "ID of the Control plane default subnet. If empty, the installer will create a subnet to use as machinesSubnet." */
+/* } */
+
+/* variable "openstack_default_machines_network_id" { */
+/*   type        = string */
+/*   default     = "" */
+/*   description = "ID of the default network for machines. If empty, the installer will create a new network." */
+/* } */
+
+/* variable "openstack_machines_subnet_ids" { */
+/*   type        = list(string) */
+/*   default     = [] */
+/*   description = "ID of each master's first subnet." */
+/* } */
+
+/* variable "openstack_machines_network_ids" { */
+/*   type        = list(string) */
+/*   default     = [] */
+/*   description = "ID of each master's first network." */
+/* } */
+
+variable "openstack_default_machines_port" {
+  type = object({
+    network_id = string
+    fixed_ips = list(object({
+      subnet_id  = string
+      ip_address = string
+    }))
+  })
```

Comment:
Is there a DRY opportunity here? Does terraform have typedefs?

Reply:
No idea.